db/migrate/20220427190212_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.string :Bio
```

Comment:
Kindly use `snake_case` as this is part of Ruby's best practice 👍 

```
      t.string : name
      t.string : photo
      t.string : bio
```

Please check for other occurrences 👍